css/modules/askQuestion.css

```diff
@@ -912,42 +819,42 @@
   right: -42px;
 }
 
-/* #virtualclassCont.congrea #askQuestion .container .footer .controls .navigation.disable {
-  pointer-events: none;
-} */
-
-/* #virtualclassCont.congrea #virtualclassAppFooterPanel #stickycontainer .chatBarTab .askQuestion_btn.highlight-new-question {
-  background: #53a83f !important;
-} */
-
 #virtualclassCont.congrea #virtualclassAppRightPanel[data-currwindow="askQuestionHeader"] #rightSubContainer #askQuestion #msgContainer {
-  background-color: #fff;
-  box-shadow: #b5b5b5 0 3px 8px;
-  padding: 0.625rem 0.125rem 0.625rem 0.625rem;
-  border-radius: 0.3125rem;
-  text-align: center;
-  z-index: 1105;
-  width: fit-content;
-  color: #9c9c9c;
-  border: 0.5px solid #e69024;
   position: absolute;
-  animation: myfirst 1s;
-  bottom: 4.5rem;
-  letter-spacing: 1.5px;
-  right: 0;
+    animation: myfirst 1s;
+    bottom: 4.5rem;
+    right: 0;
+    background-color: #ffd3b9;
+    padding: 1rem 1.5rem 1rem 1rem;
+    border-radius: 5px;
+    text-align: left;
+    z-index: 1103;
+    width: -moz-fit-content;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)